examples/llama/model.py

```diff
@@ -65,12 +65,12 @@ class ModelArgs70B(ModelArgs):
     dim: int = 8192
     n_layers: int = 80
     n_heads: int = 64
-    n_kv_heads: Optional[int] = None
+    n_kv_heads: Optional[int] = 8
     vocab_size: int = -1
     multiple_of: int = (
-        256  # make SwiGLU hidden layer size multiple of large power of 2
+        4096  # make SwiGLU hidden layer size multiple of large power of 2
     )
-    ffn_dim_multiplier: Optional[float] = None
+    ffn_dim_multiplier: Optional[float] = 1.3
```

Comment:
Where are these changes from?

Reply:
These parameters come from official llama checkpoint folder: `llama-2-70b-chat/params.json`. llama-2 read this file to overwrite the default value